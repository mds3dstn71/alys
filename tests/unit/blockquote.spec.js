import { shallowMount } from '@vue/test-utils';

import Blockquote from '@/components/blockquote/blockquote.vue';

test('renders component', () => {
  const quote = 'The quick brown fox jumps over the lazy dog.';
  const wrapper = shallowMount(Blockquote, {
    slots: {
      default: `<span>${quote}</span>`,
    },
  });

  expect(wrapper).toMatchSnapshot();
});
