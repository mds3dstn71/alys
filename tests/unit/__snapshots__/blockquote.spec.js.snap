// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `<blockquote type="multiple" class="blockquote"><span>The quick brown fox jumps over the lazy dog.</span></blockquote>`;
